### PR TITLE
Updated enumerations for posix_capability item

### PR DESCRIPTION
**Pull request by jan-cerny**

### CODE REVIEW

**Comment by mpreisler** [comment]
Discussed this with @jan-cerny . This pull request cannot be merged. It would break all OVAL < 5.11 content that uses process58 probe.

@jan-cerny will look into this. It should only export those capabilities if the content being processed is OVAL 5.11 or newer.

A test MUST be written that tests process58 with 5.10.1 AND 5.11.

File: src/OVAL/probes/unix/process58-capability.h [hunk 1]
```diff
@@ -34,5 +34,10 @@ static const struct oscap_string_map CapabilityType[] = {
 {31, "CAP_SETFCAP"},
 {32, "CAP_MAC_OVERRIDE"},
 {33, "CAP_MAC_ADMIN"},
+{34, "CAP_SYS_PACCT"},
+{35, "CAP_SYSLOG"},
+{36, "CAP_WAKE_ALARM"},
+{37, "CAP_BLOCK_SUSPEND"},
+{38, "CAP_AUDIT_READ"},
```
Comment (jan-cerny) [inline src/OVAL/probes/unix/process58-capability.h]: https://oval.mitre.org/language/version5.11/ovaldefinition/documentation/unix-definitions-schema.html#EntityStateCapabilityType

**Comment by jan-cerny** [comment]
The capabilities are now exported according to OVAL version - the new capabilities are exported only if content version is OVAL 5.11 or newer. Also I added a small test.

File: src/OVAL/probes/unix/process58-capability.h [hunk 2]
```diff
@@ -1,5 +1,7 @@
 /* map based on OVAL and /usr/include/linux/capability.h
  * not all capabilities from capability.h are allowed in OVAL */
+#define OVAL_5_10_MAX_CAP_ID 33
```
Comment (mpreisler) [inline src/OVAL/probes/unix/process58-capability.h]: Is it really OVAL 5.10? I thought these caps were added in OVAL 5.8.

File: src/OVAL/probes/unix/process58-capability.h [hunk 3]
```diff
@@ -34,5 +36,10 @@ static const struct oscap_string_map CapabilityType[] = {
 {31, "CAP_SETFCAP"},
 {32, "CAP_MAC_OVERRIDE"},
 {33, "CAP_MAC_ADMIN"},
+{34, "CAP_SYS_PACCT"},
```
Comment (mpreisler) [inline src/OVAL/probes/unix/process58-capability.h]: comments missing in the enumeration IMO, mark it clearly which caps belong where.

File: tests/probes/process58/capability.sh [hunk 4]
```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```
Comment (mpreisler) [inline tests/probes/process58/capability.sh]: license?

File: tests/probes/process58/capability.sh [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+name=$(basename $0 .sh)
+result=$(mktemp ${name}.out.XXXXXX)
+echo "result file: $result"
+stderr=$(mktemp ${name}.err.XXXXXX)
+echo "stderr file: $stderr"
+
+echo "Eval:"
+$OSCAP oval eval --results $result $srcdir/$name.oval.xml 2> $stderr
+[ -f $stderr ];
```
Comment (mpreisler) [inline tests/probes/process58/capability.sh]: Check for contents not for file existence.

File: tests/probes/process58/capability.sh [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+
+rm $stderr
+
+[ -f $result ]
```
Comment (mpreisler) [inline tests/probes/process58/capability.sh]: Can this ever fail? result is `mktemp` created.

**Pull request merged**